gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/query/TotalFetcherNode.java

```diff
@@ -0,0 +1,31 @@
+package org.hypertrace.gateway.service.entity.query;
+
+import org.hypertrace.gateway.service.entity.query.visitor.Visitor;
+
+public class TotalFetcherNode implements QueryNode {
```

Comment:
I think we should rename it to DistinctCountNode?

Reply:
That's an impl detail.